lib/stat_calculator.rb

```diff
@@ -1,15 +1,21 @@
 class StatCalculator
+
   attr_reader :games,
               :teams,
               :game_teams
 
 
+
   def initialize(games, teams, game_teams)
     @games = games
     @teams = teams
     @game_teams = game_teams
   end
 
+    def inspect
```

Comment:
Careful with your indentation here. There are times that this can cause methods to not be recognized

Reply:
Also what is this methods purpose? It doesn't seem like a statistics method

Reply:
**Answer courtesy of GPT**

"The def inspect method in Ruby is used to customize how objects appear when printed in the console or when debugging. By overriding inspect in a class, you can control what is shown when you inspect an instance of that class (e.g., by calling p object or object.inspect). This is especially helpful for cleaner, more informative debugging output, or for excluding excessive or sensitive data."

Here’s an example of how I used inspect, refactored for Ruby:
```ruby
class Person
  attr_accessor :name, :age, :military_id_number

  def initialize(name, age, military_id_number)
    @name = name
    @age = age
    @social_security_number = military_id_number
  end

  def inspect
    "#<Person: @name=#{@name.inspect}, @age=#{@age.inspect}>"
  end
end

person = Person.new("Alice", 30, "123-45-6789")
puts person.inspect
# Output: #<Person: @name="Alice", @age=30>
```

In this example, inspect omits `military_id_number`, so sensitive information isn’t displayed. This technique helps keep output manageable and relevant by including only key attributes or condensing large data structures.

If you’re working with data from CSV files in Futbol project, inspect is especially useful. Otherwise, errors or failed tests could flood your terminal with 20–30 lines of raw data. Here, regardless of the indentation, I used it specifically to simplify the output in the terminal.